### ISSUE TITLE
MJPEGStream throws NotImplementedException in .NET Core 2.0

### ISSUE THREAD

**Issue by stigvoss** [accord-net/framework#699]
Related to adding support for .NET Core #318 

MJPEGStream class throws an NotImplementedException when the SeparateConnectionGroup is set to true.
This only concerns .NET Core 2.0 and works as expected in full .NET Framework.

The source of the exception is [here](https://github.com/accord-net/framework/blob/3.6.0/Sources/Accord.Video/MJPEGStream.cs#L477), the property is seemingly not implemented in .NET Core 2.0.

The following fails with described exception:
```
MJPEGStream stream = new MJPEGStream(source)
{
    Login = ...,
    Password = ...
};
stream.NewFrame += OnFrameReceived;
stream.VideoSourceError += OnSourceError;
stream.Start();
```

The following fails at #700 
```
MJPEGStream stream = new MJPEGStream(source)
{
    Login = ...,
    Password = ...,
    SeparateConnectionGroup = false
};
stream.NewFrame += OnFrameReceived;
stream.VideoSourceError += OnSourceError;
stream.Start();
```

**Comment by stigvoss** [comment]
I seem to observe that .NET Framework supplies a ConnectStream where .NET Core supplies a WinHttpResponseStream.

**Comment by cesarsouza** [comment]
Hi @stigvoss,

Thanks a lot for reporting the issue! 

Well, I am assuming that a NotImplementedException (instead of a NotSupportedException or InvalidOperationExcept) thrown by the internal libraries of .NET Core means that some day this functionality might be implemented.

This issue seems to have been tracked in:
 - https://github.com/dotnet/corefx/issues/20043
 - https://github.com/dotnet/corefx/pull/19118

And the final resolution seems to have been to not throw exceptions, but otherwise ignore the value of this property in the connection.

I guess that we could sidestep this issue using some conditional preprocessing with ```#if NETSTANDARD2_0``` blocks.

Regards,
Cesar

**Comment by stigvoss** [comment]
I assume you are correct concerning the use of NotImplementedException instead of NotSupportedException or InvalidOperationException.

As we are dealing with a preview release of .NET Standard, this is to be expected.

To avoid breaking the code for future releases of .NET Standard, one could use a try catch statement to consume NotImplementedException for those two lines of code. Although, this will continue to have the exceptions and I assume this could impact performance. In that case, a preprocessing condition may be a valid solution for now.

If the final solution in dotnet/corefx is to ignore the input and leave out the throw clause, I assume it will have no benefit to use a try catch statement over using a preprocessing condition to leave out the code lines in the .NET Standard version.

**Comment by cesarsouza** [comment]
Included in v3.7.0.
